### ISSUE TITLE
Safe mode crash

### ISSUE THREAD

**Issue by chiefsooty1234** [zhuowei/MCPELauncher#239]
Simply if I try to turn of safe it crashes my device is acer iconia

**Comment by chiefsooty1234** [comment]
Meant blocklauncher app

**Comment by ManassehPierce** [comment]
This means there is an Issue with an installed Addon, ModPE mod, or texture pack. Try turning these off, then turning off Safe Mode.

**Comment by chiefsooty1234** [comment]
I just got the app re working today I got no add one  no texture packs no mods

**Comment by sliceofcode** [comment]
BlockLauncher does not support Acer Iconia devices with Intel Atom processors.

**Comment by Creeperface01** [comment]
Yes latest blocklauncher works on my iconia tab 8 but i can not return to normal
